Python/Wikipedia_Scraper/main.py

```diff
@@ -0,0 +1,100 @@
+from bs4 import BeautifulSoup
+import requests
+
+
+def get_player_name():
+    """ Takes Football player's name input from the user and
+    returns a string suitable to concatenate in the wikipedia URL. """
+
+    player = input('Enter the name of the Football player: ').split(' ')
+    player_name = ''
+    for i in player:
+        name = i.capitalize()
+        player_name += name
+        player_name += ' '
+
+    return player_name
+
+
+def get_club_info(headers):
+    """ Stores only the information regarding the club career of
+    a player among all the titles in the contents box. """
+
+    # For storing the club data
+    club_data = []
+
+    try:
+        start = None
+        for i in range(len(headers)):
+            each = headers[i].getText()
+            if each == "Club career":
+                start = i + 1
+                break
+
+        while headers[start].getText() != 'International career':
+            club_data.append(headers[start].getText())
+            start += 1
+
+        return club_data
+
+    # In case the name given is not a Football player
+    except TypeError:
+        print('INVALID PLAYER NAME!!')
+        return 0
+
+
+def display_info(club_data):
+    """ To display the club career information of player collected. """
+
+    for item in club_data:
+        if item[:4].isdigit():
+            print(f'    {item}')
+        else:
+            print()
+            print(f'CLUB NAME: {item}')
+
+
+def main():
+    print()
```

Comment:
A blank print statement?
If you want a new line use escape character.

Else consider removing it!